### PR TITLE
0060-feel-any-function

### PR DESCRIPTION
**Pull request by StrayAlien**

### CODE REVIEW

**Comment by opatrascoiu** [comment]
They seem fine to me.

File: TestCases/compliance-level-3/0060-feel-any-function/0060-feel-any-function.dmn [hunk 1]
```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<definitions namespace="http://www.montera.com.au/spec/DMN/0060-feel-any-function" name="0060-feel-any-function" id="_i9fboPUUEeesLuP4RHs4vA" xmlns="http://www.omg.org/spec/DMN/20180521/MODEL/" xmlns:di="http://www.omg.org/spec/DMN/20180521/DI/" xmlns:dmndi="http://www.omg.org/spec/DMN/20180521/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <description>FEEL built-in function 'any(list)' in category list functions</description>
+    <decision name="decision001" id="_decision001">
+        <description>Tests FEEL expression: 'any([true, false, true])' and expects result: 'false (boolean)'</description>
+        <question>Result of FEEL expression 'any([true, false, true])'?</question>
+        <allowedAnswers>false (boolean)</allowedAnswers>
```
Comment (tarilabs) [inline TestCases/compliance-level-3/0060-feel-any-function/0060-feel-any-function.dmn]: minor tpoe, I believe here is `true`

File: TestCases/compliance-level-3/0060-feel-any-function/0060-feel-any-function.dmn [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+        <variable typeRef="boolean" name="decision004"/>
+        <literalExpression>
+            <text>any([123, false])</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision005" id="_decision005">
+        <description>Tests FEEL expression: 'any([])' and expects result: 'true (boolean)'</description>
+        <question>Result of FEEL expression 'any([])'?</question>
+        <allowedAnswers>true (boolean)</allowedAnswers>
```
Comment (tarilabs) [inline TestCases/compliance-level-3/0060-feel-any-function/0060-feel-any-function.dmn]: minor typo, I believe here is `false`.

**Comment by agilepro** [comment]
Please address typos and we will merge

**Comment by StrayAlien** [comment]
@agilepro . typos fixed.

**Pull request merged**